### ISSUE TITLE
Updating stack doesn't confirm anything is happening

### ISSUE THREAD

**Issue by robmarcer** [flowforge/flowforge#2364]
### Current Behavior

When there is a new stack available, an update button is shown in the instance overview page.

If that is clicked it makes you to the instances settings page, if I then click the button to update stack I'm given no confirmation anything is happening.

This resulted in my trying a couple of times before I concluded that it was actually updating but no message had confirmed that.

### Expected Behavior

I should be told the stack is updating so that I don't try again.

### Steps To Reproduce

Go to instance overview for an instance on a legacy stack.

Click update

On the next page click update stack.

### Environment

- FlowForge version: 1.8 (FF Cloud)
- Node.js version: NA
- npm version: NA
- Platform/OS: NA
- Browser: NA

### Have you provided an initial effort estimate for this issue?

I can not provide an initial effort estimate

**Comment by knolleary** [comment]
Where did you see this? In production/staging/local?

There is an open issue with our 'loading' animations not showing properly in the current development version (local/staging) - #2354 - which could be a cause of the issue here. That said, you ought to get a toast notifation in the top right when the update is complete.

There *is* a bug to be fixed that after the update completes it takes you back to the overview page, but the 'update' button is still there... something isn't quite refreshing properly.

If you tell me this was on Production, then something else is going on...

**Comment by robmarcer** [comment]
Hello, this was production.

Sorry I said 1.9 when it's actually 1.8, I'll update the issue.

**Comment by robmarcer** [comment]
This shows the issue - https://d.pr/v/5VetVw

To be fair, it's possible I missed the confirmation pop up.

**Comment by knolleary** [comment]
I just tried to recreate in production. I got a fleeting bit of feedback, before being redirected to the overview page. But the 'update' button remained and there was no feedback the app was restarting.

In fact, as I type, you've shared a video that shows the same - which is good news. I've identified the cause of why the overview page doesn't refresh and have raised a PR that fixes that bit - #2371
